content/codespaces/managing-your-codespaces/managing-gpg-verification-for-codespaces.md

```diff
@@ -27,3 +27,6 @@ Once you enable GPG verification, it will immediately take effect for all your c
   ![Radio buttons to manage GPG verification](/assets/images/help/settings/codespaces-gpg-verification-radio-buttons.png) 
 1. If you chose "Selected repositories", select the drop-down menu, then click a repository you want enable GPG verification for. Repeat for all repositories you want to enable GPG verification for.
   !["Selected repositories" drop-down menu](/assets/images/help/settings/codespaces-gpg-verification-repository-drop-down.png) 
+
+
+> Note: once you have enabled this setting, ensure the "Git: Always Sign Off" and "Git: Enable Commit Signing" are enabled from the Visual Studio Code interface.
```

Comment:
```suggestion
**Note:** once you have enabled this setting, ensure the "Git: Always Sign Off" and "Git: Enable Commit Signing" are enabled from the Visual Studio Code interface.
```

Reply:
@ramyaparimi Thanks for the feedback, Like this?

Reply:
@baywet Like this :) Please checkout [callout-tags](https://github.com/github/docs/blob/main/contributing/content-markup-reference.md#callout-tags) for more information 🐬